rb-files/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise not_implemented_error
+  end
```

Comment:
- [ ] To correctly match the project requirements, kindly make sure you update your `correct_name` method as right now it is showing an error of camel_case notation. You can change it to something like this below by also adding the message to let users know what the whole error is about.
```ruby
class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```